src/FlashMessage/FlashMessage.tsx

```diff
@@ -42,55 +42,43 @@ const FlashMessage = ({ flashMessage }: FlashMessageProps) => {
     }
 
     const close = () => {
-        if (timeoutHandler.current) {
-            clearTimeout(timeoutHandler.current);
-        }
+        clearTimeout(timeoutHandler.current);
         dispatch(removeMessage(id));
     };
 
     const addFadeout = () => {
-        if (dismissTime != null) {
-            timeoutHandler.current = setTimeout(() => {
-                dispatch(removeMessage(id));
-            }, dismissTime);
+        if (dismissTime) {
+            timeoutHandler.current = setTimeout(
+                () => dispatch(removeMessage(id)),
+                dismissTime
+            );
+            setFadeoutTimer(`${dismissTime}ms`);
         }
-        setFadeoutTimer(`${dismissTime}ms`);
     };
 
     const removeFadeout = () => {
         setFadeoutTimer('unset');
-        if (timeoutHandler.current) {
-            clearTimeout(timeoutHandler.current);
-        }
+        clearTimeout(timeoutHandler.current);
     };
 
-    const initialRender = () => divRef.current == null;
-
     return (
         <div
             ref={divRef}
             style={{
                 backgroundColor: getBackgroundColorFromVariant(variant),
                 color: colors.white,
                 zIndex: 1000,
-                animation: initialRender() ? 'slide-in 1s' : 'unset',
+                animation: !divRef.current ? 'slide-in 1s' : 'unset',
```

Comment:
I think the function name alone is a good enough reason to keep the function, as it gives some explanation to why you want `divRef.current` to be `null` or `undefined` in order to add the animation. This may not be trivial coming back to the code in 1 week time.